Entity/CategoryManager.php

```diff
@@ -97,6 +97,32 @@ public function getSubCategoriesPager($categoryId, $page = 1, $limit = 25, $crit
     }
 
     /**
+     * @param CategoryInterface $category
+     *
+     * @return CategoryInterface
+     */
+    public function getRootCategoryWithChildren($category)
```

Comment:
@greg0ire, One of them is used in tests, and I plan to bring tests for the rest. And PHPUnit cannot mock final methods ;(